packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -142,6 +142,10 @@ export const messages = {
     'No function.toml found. Create a function.toml, or create a new function using SFDX: Create Function.',
   force_function_start_warning_not_in_function_folder:
     'Open a function file to run SFDX: Start Function',
+  force_function_start_warning_plugin_not_installed:
+    'To run this command, install the evergreen plugin. For more info, see [Getting Started with Salesforce Functions](https://github.com/forcedotcom/evergreen-docs/blob/master/Getting%20Started%20Guide/Evergreen%20Getting%20Started%20Guide.md#installing-the-salesforce-functions-sfdx-plugin).',
+  force_function_start_warning_docker_not_installed_or_not_started:
+    'It looks like Docker is not installed or running. To run this command, install and start Docker Desktop from [https://www.docker.com/products/docker-desktop](https://www.docker.com/products/docker-desktop)',
```

Comment:
@ekapner @danyusf These are the messages shown to users in VS Code when evergreen plugin is not installed or docker is not installed or started.

- Is there a better doc website URL for installing the evergreen plugin?
- Does the official docker desktop link https://www.docker.com/products/docker-desktop work?

Reply:
Because Evergreen isn't GA yet (and won't be until roughly the 232 time-frame), that Getting Started doc link is correct. Note that that link is in a private github repo that we add pilot customers to, so anyone getting this message who's not in the pilot program will get a 404 for that link (but, truthfully, it would be unexpected to have a customer get this message who's not already working with the pilot).

I believe that Docker link should work (it's the same link I use in the Evergreen dev guide).

One request (which I probably should have mentioned earlier) -- avoid using the term "Evergreen", as product marketing has decided the product name will be "Salesforce Functions", and the Evergreen term will be mostly an internal term. @ekapner I believe previous PRs I've looked at were not using "Evergreen", but if I missed any uses, my apologies.

Reply:
@danyusf - thanks for the reminder about Evergreen - and I believe you're correct that I haven't seen it used in previous messages we've reviewed.

Reply:
@danyusf @ekapner  Thank you for the reminder - Yes I haven't mentioned evergreen other than this one. I'll update it to "Salesforce Functions"

Reply:
@xyc - I installed Docker from that link